src/components/EduIDTextInput.js

```diff
@@ -46,34 +46,32 @@ const textInput = props => {
   }
 
   let field;
-
-  if (type === "select") {
-    let options = [];
-    if (selectOptions) {
-      options = selectOptions.slice();
-    }
-    const children = options.map(opt => {
+  if (selectOptions) {
```

Comment:
very minor thought, but it struck me now that (because this used to be a select html tag in the past and is not anymore) maybe we should be sure to clean out any mentions of select and make it really clear that it is the code for the new radio buttons

Reply:
I understan why you think it make you confused. its a kind of trick, some reason can't  have a type prop in Field that cause an error (can not check any radio button) but we need a check(`if (selectOptions)`) for different type of input rendering. however, once we have separate name and language functions then we don't need the `if (selectOptions)` anymore.